site/content/en/docs/concepts/cluster_queue.md

```diff
@@ -60,6 +63,11 @@ Kueue assigns the first flavor in the ClusterQueue's `.spec.resourceGroups[*].fl
 list that has enough unused `nominalQuota` quota in the ClusterQueue or the
 ClusterQueue's [cohort](#cohort).
 
+Since `pods` resource name is [reserved](/docs/conceps/workload#reserved-resource-names) and it's value
+is computed by Kueue in the during [admission](/docs/concepts#admission), not provided by the [batch user](/docs/tasks/#batch-user),
+it could be used by the [cluster administrators](/docs/tasks#batch-administrator) to limit the number of zero or very
```

Comment:
```suggestion
it could be used by the [batch administrators](/docs/tasks#batch-administrator) to limit the number of zero or very
```

nit: Consistently using the `batch administrators` would be good throughout the whole of the docs.

Reply:
done